server.js

```diff
@@ -89,3 +92,64 @@ os_info_data = JSON.stringify(os_info_data)
 
 
 }
+
+const server = http.createServer((request, response) => {
+  // Console.log any error that occurs when the request was made
+  request.on('error', (err) => {
+    console.log('An error ocurred')
+  })
+
+  // Console.log any error that occurs when the response was made
+  response.on('error', (err) => {
+    console.log('An error ocurred')
+  })
+
+  // set request.url to const urlPath for fast access
+  const urlPath = request.url
+
+  // Check if the urlPath == '/'. If true, load the index.html page
+  if (urlPath == '/')
+  {
+    response.writeHead(200, {'Content-Type': 'text/html'})
+    const data = fs.readFileSync('./pages/index.html')
```

Comment:
# Use read file with a callback 
# Implement error checking for if the file returns an error